Add system-information extension

Visual-Studio-Coder opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

Visual-Studio-Coder commented:
@pernielsentikaer I just was wondering if there was a way I could have an action called "open system settings" and then open that about this mac thing, but I'm a bit unsure how to do it. Please give me some advice on doing this. Thanks in advance!

pernielsentikaer commented on extensions/system-information/src/index.jsx:
```diff
@@ -0,0 +1,106 @@
+    });
+  }, []);
+
+  const releaseImage = () => {
```
You forgot about Sonoma, and it would be cool if images were added to assets instead.

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

I think you can do something like this (haven't tested)

```
actions={<ActionPanel><Action.Open target="x-apple.systempreferences:com.apple.Accessibility-Settings.extension" title="Open" /></ActionPanel>}
```

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

Visual-Studio-Coder commented:
@pernielsentikaer I made some changes, and when I tried npm run publish, I got this
```
> publish
> npx @raycast/api@latest publish

ready -  validate git repository
ready -  getting fork
wait  -  preparing clone...failed running git command 'git checkout main', exit status 1

error: Your local changes to the following files would be overwritten by checkout:
        extensions/system-information/src/index.jsx
Please commit your changes or stash them before you switch branches.
Aborting
```

Please tell me how to fix this, but if not, then could you please just try to merge the repo I've already made (and updated) for me?

Here is the link to that repo: https://github.com/Visual-Studio-Coder/system-information

pernielsentikaer commented:
Did you try to run `npx @raycast/api@latest pull-contributions`

Visual-Studio-Coder commented:
Okay, some nice staff member helped me get this fixed, so shoutout to mathieudutour. I have changed the icon and added some more actions, like checking storage and updates.

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
